examples/projects/Manhattan/openfluid-project.conf

```diff
@@ -5,3 +5,6 @@ Authors=Michael Rabotin
 IncOutput=false
 CreationDate=20130424T085754
 LastModDate=20130424T085754
+
+[builder.model.graphicalview]
+traffic.surf.car-transfer=@Point(8 244)
```

Comment:
Where is the placement of the `traffic.surf.trafficlights-state` simulator? To be checked